src/main/telemetry/ghst.c

```diff
@@ -213,6 +217,32 @@ typedef enum {
 static uint8_t ghstScheduleCount;
 static uint8_t ghstSchedule[GHST_SCHEDULE_COUNT_MAX];
 
+static bool mspReplyPending;
+
+void ghstScheduleMspResponse()
+{
+    mspReplyPending = true;
+}
+
+static void ghstSendMspResponse(uint8_t *payload, const uint8_t payloadSize)
+{
+    sbuf_t ghstPayloadBuf;
+    sbuf_t *dst = &ghstPayloadBuf;
+
+    static uint8_t mspFrameCounter = 0;
+    DEBUG_SET(DEBUG_GHST_MSP, 1, ++mspFrameCounter);
+
+    ghstInitializeFrame(dst);                                                           // addr
+    sbufWriteU8(dst, GHST_PAYLOAD_SIZE + GHST_FRAME_LENGTH_CRC + GHST_FRAME_LENGTH_TYPE);     // lenght
+    sbufWriteU8(dst, GHST_DL_MSP_RESP);                                                 // type
+    sbufWriteData(dst, payload, payloadSize);                                           // payload
+    for(int i = 0; i < GHST_PAYLOAD_SIZE - payloadSize; ++i)                            // payload fill zeroes
+    {
```

Comment:
missing space. bracket should be on line above

Reply:
Thank you, fixed.